### ISSUE TITLE
[TalkBack] [Upstream] "Navigate Up" button in TalkBack settings doesn't work

### ISSUE THREAD

**Issue by PatrykMis** [GrapheneOS/os-issue-tracker#1486]
# Steps to reproduce

Enter TalkBack Settings via Settings -> Accessibility -> TalkBack or via TalkBack menu, choose "Sound and Vibration", "Verbosity" or any of the fragments and press "Navigate Up" button on upper left corner.

# Notes

The button doesn't work, the only way to navigate to the previous screen is to press Back button on the 3-button navigation bar or via gesture in gesture navigation.

I have built debug version (under different app ID), enabled verbose output in developer settings but I found nothing interesting in the logcat output which I attached.

I'm beginner, anybody would like to help me or solve this problem? Don't know where is the code responsible for preference fragments navigation in TalkBack.

[logcat_tb.txt](https://github.com/GrapheneOS/os-issue-tracker/files/9577278/logcat_tb.txt)

**Comment by thestinger** [comment]
We lack the resources to develop this.
